![](https://raw.githubusercontent.com/evohologen/.github/refs/heads/main/profile/ceh_logo.png)

# Welcome to the Center for Evolutionary Hologenomics' code hub

In this organisational repository you will find the links and references to the projects and studies conducted by CEH researchers.

## 2025

### Pietroni et al. 2025

Hologenomic data generation and analysis in wild vertebrates

**Methods in Ecology and Evolution 16: 97-107**

:octocat: https://github.com/earthhologenome/EHI_technical_considerations

:book: https://www.earthhologenome.org/EHI_technical_considerations/

### Aizpurua et al. 2025

Functional insights into the effect of feralisation on the gut microbiota of cats worldwide

**Molecular Ecology 34: e17695**

:octocat: https://github.com/alberdilab/domestic_feral_cat_metagenomics

:book: https://alberdilab.github.io/domestic_feral_cat_metagenomics

### Hernández et al. 2025

Contrasting recovery of metagenome‑assembled genomes and derived bacterial communities and functional profiles from lizard fecal and cloacal samples

**Animal Microbiome 7: 15**

:octocat: https://github.com/alberdilab/domestic_feral_cat_metagenomics

:book: https://alberdilab.github.io/domestic_feral_cat_metagenomics

## 2024

### Thormar et al. 2024a

Sampling fish gut microbiota - A genome-resolved metagenomic approach

**Ecology and Evolution 14(9): e70302**

:octocat: https://github.com/alberdilab/microbiota_adaptation_review

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB71469

### Katirtzoglou et al. 2024a

Intestinal microbial profiles of wild Alaskan rainbow trout (Oncorhynchus mykiss) characterized by 16S rRNA amplicon data

**Data in Brief 57: 110902**

:octocat: https://github.com/alberdilab/microbiota_adaptation_review

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB71469

### Martin-Bideguren et al. 2024

Quantitative synthesis of microbe-driven acclimation and adaptation in wild vertebrates

**Evolutionary Applications 17(10): e70025**

:octocat: https://github.com/alberdilab/microbiota_adaptation_review

### Brealey et al. 2024

Host–gut microbiota interactions shape parasite infections in farmed Atlantic salmon

**mSystems 9(2): e01043-23**

:octocat: https://github.com/jcbrealey/Brealey_etal_salmon_multiomics

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB64334

### Eisenhofer et al. 2024a

Quantifying microbial DNA in metagenomes improves microbial trait estimation

**ISME Communications 4(1): ycae111**

:octocat: https://github.com/alberdilab/domestic_feral_cat_metagenomics

### Thormar et al. 2024b

A zebrafish model to elucidate the impact of host genes on the microbiota

**Environmental DNA 6(1): e513**

:octocat: https://github.com/eirikurandri/ZEBRAFISH-AND-CRISPR-CAS-A-MODEL-TO-ELUCIDATE-HOST-GENETIC-EFFECTS-ON-THE-MICROBIOTA

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB51322

### Katirtzoglou et al. 2024b

Genomic context determines the effect of DNA methylation on gene expression in the gut epithelium of Atlantic salmon (Salmo salar)

**Epigenetics 19: 2392049**

:octocat: https://github.com/katkat-ri/EpiG-salmon-2024

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB64334

### Eisenhofer et al. 2024b

A comparison of short-read, HiFi long-read, and hybrid strategies for genome-resolved metagenomics

**Microbiology spectrum 12(4): e03590**

:octocat: https://github.com/alberdilab/domestic_feral_cat_metagenomics

## 2023

### Hansen et al. 2023

Intestinal epigenotype of Atlantic salmon (Salmo salar) associates with tenacibaculosis and gut microbiota composition

**Genomics 115(3): 110629**

:octocat: https://github.com/SrenBlikdal/Tenacibaculosis

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB59345

### Rasmussen et al. 2023

Co-diversification of an intestinal Mycoplasma and its salmonid host

**ISME 17: 682–692**

:octocat: https://github.com/JacobAgerbo/Norwegian_wild_Salmon_2022

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB59886

## 2022

### Rasmussen et al. 2022a

Integrative analyses of probiotics, pathogenic infections and host immune response highlight the importance of gut microbiota in understanding disease recovery in rainbow trout (Oncorhynchus mykiss)

**Journal of Applied Microbiology 132(4):3201-3216**

:octocat: https://github.com/JacobAgerbo/Disease_Challenge_Rainbow_Trout

:file_folder:	https://www.ebi.ac.uk/ena/browser/view/PRJEB48694

### Nyholm et al. 2022

Gut microbiota differences between paired intestinal wall and digesta samples in three small species of fish

**PeerJ 10: e12992**

:octocat: https://github.com/LasseNyholm/Gut-microbiota-differences-between-paired-mucus-and-digesta-samples-in-three-small-species-of-fish

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB48573

### Brealey et al. 2022

Microbiome “Inception”: an Intestinal Cestode Shapes a Hierarchy of Microbial Communities Nested within the Host

**mBio e00679-22**

:octocat: https://github.com/jcbrealey/cestode_microbiome

:file_folder:	https://www.ebi.ac.uk/ena/browser/view/PRJEB51496

### Rasmussen et al. 2022b

A multi-omics approach unravels metagenomic and metabolic alterations of a probiotic and synbiotic additive in rainbow trout (Oncorhynchus mykiss)

**Microbiome 10: 21**

:octocat: https://github.com/JacobAgerbo/Multi_Omic_Rainbow_Trout

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB48695

## 2021

### Rasmussen et al. 2021

Genome-resolved metagenomics suggests a mutualistic relationship between Mycoplasma and salmonid hosts

**Communications Biology 4: 579**

:octocat: https://github.com/JacobAgerbo/Comparative_Mycoplasma

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJEB40990

### Bozzi et al. 2021

Salmon gut microbiota correlates with disease infection status: potential for monitoring health in farmed animals

**Animal Microbiome 3: 30**

:octocat: https://github.com/DavideBozzi/Bozzi_et_al_2020_analysis

:file_folder: https://www.ebi.ac.uk/ena/browser/view/PRJNA665207
